lib/taskbook.js

```diff
@@ -111,13 +110,13 @@ class Taskbook {
       }
     });
 
-    const description = desc.join(' ');
+    const descriptions = desc.join(' ').split('::');
```

Comment:
Maybe just add this as a constant or even an option somewhere to be changed easily.